lib/src/components/NxForm/NxForm.scss

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-form__required-field-notice {
+    display: block;
```

Comment:
> The original intent was that this element be an inline element that gets placed into a container that gives it its block layout and also its text styling (e.g. bold, dark). That intent may or may not hold as you work out how this element should be wrapped when not within `NxTile.HeaderSubtitle`, but for now these styles don't seem right

When discussing with Design, we agreed on a margin-bottom of 24px. In order to do so, my solution at the time was to use display:block to add the margin and collapse the margin. I'm trying to find a selector that would target the container element when it holds .nx-form_required-field-notice. I see :has (https://developer.mozilla.org/en-US/docs/Web/CSS/:has),  but it's very unsupported. Do you have any recommendations of how to target the container element to be styled only when it contains .nx-form_required-field-notice, that doesn't involve adding classes?

Reply:
> When discussing with Design, we agreed on a margin-bottom of 24px

That implies that design is abandoning the approach of having the message appear in an existing "slot" such as the tile subheader.  Is that the case? Tile subheaders only have 16px of bottom spacing for instance.

> I'm trying to find a selector that would target the container element when it holds .nx-form_required-field-notice. I see :has (https://developer.mozilla.org/en-US/docs/Web/CSS/:has), but it's very unsupported. Do you have any recommendations of how to target the container element to be styled only when it contains .nx-form_required-field-notice, that doesn't involve adding classes?

No we'll make bigger changes if that's the direction we want to go.

Reply:
> > When discussing with Design, we agreed on a margin-bottom of 24px
> 
> That implies that design is abandoning the approach of having the message appear in an existing "slot" such as the tile subheader. Is that the case? Tile subheaders only have 16px of bottom spacing for instance.
> 

At the time of discussion, we had liked the bolded version with a 24px bottom and keeping it the same in all instances.I remember we had a discussion to the reasoning about 24px bottom, although what we talked about exactly escapes me now. 
 I did not bring it up that it would mean having to create a new "slot" instead of an existing one, so that is my fault.  Is creating a new "slot" an option, or would you rather we stay within pre-existing elements and styles? (Also, I am trying to locate subheaders, and only see a sublabel. Where is subheader located?)

Reply:
> If that latter approach is what we are now going with, then _all_ of the styling of the Required Field Notice is now up to itself. Whereas the current implementation relies on the wrapper (e.g. the Tile Subtitle) to provide the vertical spacing and font weight, in this different approach all of that would be built into the Required Field Notice component. Going along with that, it would be changed from a `span` to a `p`, making it a block element.
> 

After speaking to Jason, they believe that in the given gallery examples (and all scenarios they can think of), leaving the Required Field text as part of the content of the form makes the most sense. However, Jason did say they would like to leave the option for someone to use it inside a header as well, which is why I still wrapped the element in a <p> tag instead of updating and directly styling the <span> element.

Reply:
As discussed, NxForm.RequiredFieldNotice has been updated- the span has been replaced with a paragraph element: [6a6aa83](https://github.com/sonatype/sonatype-react-shared-components/pull/735/commits/6a6aa8356c243bf8f7437c442254114bbc3209b0)